tests/notebooks/output.ipynb

```diff
@@ -16,8 +16,9 @@
        "Output()"
       ]
      },
+     "execution_count": 1,
      "metadata": {},
-     "output_type": "display_data"
+     "output_type": "execute_result"
```

Comment:
Since the diff also includes formatting changes, wondering if this switch of `output_type` is the one that did the trick?

Reply:
Yes, since we are testing against `ipywidgets 8`, the output of the widget is now of type `execute_result` instead of `display_data`.

Reply:
Nice thanks :+1: